tests/php/FileTest.php

```diff
@@ -801,7 +801,7 @@ public function testRename()
     public function testGetAllowedExtensions($allowedExtensions, $expected)
     {
         Config::modify()->set(File::class, 'allowed_extensions', $allowedExtensions);
-        $this->assertSame($expected, $expected);
```

Comment:
This is one of the funnier things I've ever had to fix in a PR 😆

Reply:
:see_no_evil: